src/coreclr/jit/emitarm64.cpp

```diff
@@ -15668,10 +15892,6 @@ void emitter::emitDispArrangement(insOpts opt)
         case INS_OPTS_SCALABLE_B_WITH_PREDICATE_MERGE:
             str = "b";
             break;
-        case INS_OPTS_4H:
```

Comment:
does removing these have any implications on existing instructions that we added so far? Can you try to compare the output of all instructions jitdisasm vs. cstool with this change?

Reply:
> Can you try to compare the output of all instructions jitdisasm vs. cstool with this change?

Sure thing. I'll post a comment with diffable outputs from both.